app/services/WhatYouOweService.scala

```diff
@@ -79,18 +79,21 @@ class WhatYouOweService @Inject()(val financialDetailsService: FinancialDetailsS
           chargesList = getFilteredChargesList(financialDetailsModelList, isReviewAndReconciledEnabled, isFilterCodedOutPoasEnabled),
           codedOutDocumentDetail = codedOutChargeItem)
 
-        callOutstandingCharges(mtdUser.saUtr, mtdUser.incomeSources.yearOfMigration, dateService.getCurrentTaxYearEnd).map {
+        callOutstandingCharges(dateService.getCurrentTaxYearEnd).map {
           case Some(outstandingChargesModel) => whatYouOweChargesList.copy(outstandingChargesModel = Some(outstandingChargesModel))
           case _ => whatYouOweChargesList
         }
     }
   }
 
-  private def callOutstandingCharges(saUtr: Option[String], yearOfMigration: Option[String], currentTaxYear: Int)
-                                    (implicit headerCarrier: HeaderCarrier): Future[Option[OutstandingChargesModel]] = {
+  private def callOutstandingCharges(currentTaxYear: Int)
```

Comment:
If we can use next method signature instead =>
`callOutstandingCharges(currentTaxYear: TaxYear)`
In general, we are planning to use TaxYear type everywhere in method signatures and extend TaxYear type with relevant operation like: compare two taxYears etc